app/src/main/java/edu/uc/hickmadc/customnotifications/service/GroupService.kt

```diff
@@ -19,9 +19,9 @@ class GroupService {
     fun getAllGroups() : ArrayList<Group>{
         val groups : ArrayList<Group> = ArrayList()
         //placeholder, should be replaced by database call/live data
-        groups.add(Group(name = "Group1", desc = "This data is", active = true, same_schedule = false, groupId = 1))
-        groups.add(Group(name = "Group2", desc = "set in the", active = false, same_schedule = false, groupId = 2))
-        groups.add(Group(name = "Group3", desc = "main view model", active = true, same_schedule = false, groupId = 3))
+        groups.add(Group(name = "Group1", desc = "This data is", active = true, same_schedule = false))//, groupId = 1))
```

Comment:
Commented out group Id because project didn't recognize it (wasn't in parameter)

Reply:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.